Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -175,105 +173,6 @@ public void BadgeTest ()
       Helper.RunScreenshotTestExact<IFluentScreenshotElement<ElementScope>, ScreenshotTest>(element.ForElementScopeScreenshot(), ScreenshotTestingType.Both, test);
     }
 
-    /// <summary>
-    /// Tests that the <see cref="ScreenshotTooltipAnnotation"/> is drawn correctly.
-    /// </summary>
-    [Category("Screenshot")]
-    [Test]
-    public void DrawTooltip ()
```

Comment:
What is DrawTooltip and DrawTooltip_WithFollowingBoxAnnotation doing that require a UI? Don't we also have tooltips manually drawn for headless tests?